example/code_systems_translations/EncounterClassDE.json

```diff
@@ -0,0 +1,103 @@
+{
+  "resourceType": "CodeSystem",
```

Comment:
the translations should not be checked in ->
delete all translations from the folder.
add Readme to folder which states the purpose of the folder
add example/code_systems_translations/*.json to gitignore
ensure that you only read in .json files from the folder when loading the translations